### PR TITLE
Add file links to Table Of Contents

### PR DESCRIPTION
**Pull request by barbaraperic**
Add `Contract automation` and `Relay` file to SUMMARY.md

[Related to this issue ](https://github.com/filecoin-project/filecoin-docs/pull/2324).

### CODE REVIEW

File: smart-contracts/advanced/aggregated-deal-making.md [hunk 1]
```diff
@@ -55,12 +55,12 @@ To request for aggregation and PoDSI off-chain, developers interact with an aggr
 
 On-chain aggregation and PoDSI requests go through aggregator oracle smart contracts:
 
-1. The client [prepares the data](../../storage-providers/filecoin-deals/storage-deals/#data-preparation) and generates the sub-piece CID, known as pCID (CommPc). Here is an easy [data preparation tool](https://data.lighthouse.storage/) by [lighthouse.storage](https://lighthouse.storage).
+1. The client [prepares the data](https://github.com/filecoin-project/filecoin-docs/blob/main/storage-providers/filecoin-deals/storage-deals.md#data-preparation) and generates the sub-piece CID, known as pCID (CommPc). Here is an easy [data preparation tool](https://data.lighthouse.storage/) by [lighthouse.storage](https://lighthouse.storage).
```
Comment (mirhamasala) [inline smart-contracts/advanced/aggregated-deal-making.md:58]: Is there a reason these need to be absolute now?

Comment (barbaraperic) [reply smart-contracts/advanced/aggregated-deal-making.md:58]: The links are currently not working on [filecoin docs](https://docs.filecoin.io/smart-contracts/advanced/aggregated-deal-making), so I changed them to absolute. I haven't investigated further, but I can dig in?

Comment (mirhamasala) [reply smart-contracts/advanced/aggregated-deal-making.md:58]: Hmm .. I not sure why not .. the path seems correct. 😕

Comment (barbaraperic) [reply smart-contracts/advanced/aggregated-deal-making.md]: Thank you, fixed!

**Review by mirhamasala — APPROVED** [review]
🙌🏼✨ Thank you for fixing this! (CC: @trruckerfling)

**Pull request merged**